From 0b5812b96f8e7b0c2072a682205109ab65ef336f Mon Sep 17 00:00:00 2001
From: Firoz Khan <firoz.khan@linaro.org>
Date: Mon, 7 Jan 2019 20:12:06 +0530
Subject: [PATCH 07/27] c6x: add __ARCH_NOMMU as a bugfix

Add __ARCH_NOMMU in uapi/asm/unistd.h file as a bugfix.

Signed-off-by: Firoz Khan <firoz.khan@linaro.org>
---
 arch/c6x/include/uapi/asm/unistd.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/c6x/include/uapi/asm/unistd.h b/arch/c6x/include/uapi/asm/unistd.h
index 6b2fe79..9e9ffe5 100644
--- a/arch/c6x/include/uapi/asm/unistd.h
+++ b/arch/c6x/include/uapi/asm/unistd.h
@@ -18,6 +18,7 @@
 #define __ARCH_WANT_RENAMEAT
 #define __ARCH_WANT_STAT64
 #define __ARCH_WANT_SYS_CLONE
+#define __ARCH_NOMMU
 
 /* Use the standard ABI for syscalls. */
 #include <asm-generic/unistd.h>
-- 
1.9.1

